### DISCUSSION TITLE
how to combine two apps of joint js+

### DISCUSSION THREAD

**Discussion by murmuhimanshu** [clientIO/joint#2310] [Q&A]
### Introduction

I am trying to combine two apps of jointjs+. Basically want to use the feature of the kitchen sink app with expand/collapse app, is there a way to use both the application? Also can we combine more than 2 apps?
Is there any resources or a demo to implement that

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Each application is just an example of how to combine JointJS plugins to create different products. You can combine any number of samples to create another unique application. I recommend that you start by understanding the individual plugins and their configuration, and then you can try to integrate one app into the other (or write a new app inspired by the 2 apps from scratch).

Tutorial: https://resources.jointjs.com/tutorial/getting-started
Docs: https://resources.jointjs.com/docs/rappid/v3.7/index.html
